tests/components/history_stats/test_sensor.py

```diff
@@ -1366,10 +1477,6 @@ def _fake_states(*args, **kwargs):
 
     past_next_update = start_time + timedelta(minutes=30)
     with (
-        patch(
```

Comment:
This step in the test no longer hits the recorder, so does not need to be patched.